include/LeMonADE/feature/FeatureConnectionSc.h

```diff
@@ -56,12 +59,29 @@ class MonomerReactivity
 	//! Getting the number of maximum possible bonds for the monomer.
 	uint32_t getNMaxBonds() const {return nMaxBonds;};
 
+	MonomerReactivity getReactivity(){return *this; }
+	void setReactivity(const MonomerReactivity& react)
```

Comment:
bad practice to call a member function not by its name -> rename to getMonomerReactivity() and setMonomerReactivity()